guestfs_generator/tests/test_guestfs_generator.py

```diff
@@ -0,0 +1,18 @@
+"""
```

Comment:
More tests would be very appreciated, but the real test is if this workflow runs. Not much else matters right now.